serverless/admin_guide/serverless-cluster-admin-eventing.adoc

```diff
@@ -0,0 +1,19 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-cluster-admin-eventing"]
+= Creating Eventing components as a cluster administrator
+include::modules/common-attributes.adoc[]
+:context: serverless-cluster-admin-eventing
+
+toc::[]
+
+If you have cluster administrator permissions on a {product-title} cluster, you can complete various tasks with {ServerlessProductName} in the *Administrator* perspective of the web console and by using the `kn` and `oc` CLIs.
```

Comment:
Can you be a little more specific about the tasks in the introduction for both this assembly and the next?

Reply:
Updated a bit, hopefully that helps, but I don't want to call out specific tasks here since things still need to be added and it will create more maintenance work long term. This PR is mostly to add the structure and then I will be adding more modules on serving and eventing tasks in the web console etc. after.

Reply:
This does sound better!